src/SonarScanner.MSBuild.Shim/SonarScanner.MSBuild.Shim.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(ScannerNetStandardVersion);$(ScannerNetFxVersion)</TargetFrameworks>
+    <AssemblyName>SonarScanner.MSBuild.Shim</AssemblyName>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\SonarScanner.MSBuild.Common\SonarScanner.MSBuild.Common.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
Most of the modified project files have this resources-related item group. Didn't you say that it wasn't strictly necessary, but that VS keeps re-adding it?

Reply:
Correct. Do you think we should keep these changes (so we don't have to keep fight VS), or remove (so it's cleaner). I haven't decided which option I prefer, WDYT?

Reply:
Last time you said you preferred to delete them ;-)
However, if it doesn't make any difference in practice then I'd leave it in so we don't have to keep removing it.

Reply:
I did. Guess I'm getting tired of removing that over and over :)